debs/SPECS/3.11.0/wazuh-api/debian/control

```diff
@@ -0,0 +1,12 @@
+Source: wazuh-api
+Section: admin
+Priority: extra
+Maintainer: Wazuh, Inc <info@wazuh.com>
+Build-Depends: debhelper (>= 7.0.50~), nodejs (>= 4.6), npm, curl, gawk
+Standards-Version: 3.8.4
+Homepage: http://www.wazuh.com
+
+Package: wazuh-api
+Architecture: any
+Depends: ${shlibs:Depends}, npm, nodejs (>= 4.6.1), wazuh-manager (>= 3.10.0), wazuh-manager (<< 3.10.1), curl
```

Comment:
The depends lines are wrong. It must lock the dependency with wazuh-manager 3.11.0

Reply:
Done in https://github.com/wazuh/wazuh-packages/pull/254/commits/bd9469d9104aaced5b40e1a09a527bbe1cefa707

Reply:
This isn't resolved yet.